odk1-src/collect-connect.rst

```diff
@@ -33,13 +33,34 @@ Configure server from QR code
 
 .. _other-collect-server-options:
 
+Configure server from settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. From the Action Button (:guilabel:`⋮`), select :menuselection:`General Settings`
+
+
+   .. image:: /img/collect-connect/main-menu-highlight-kebab.*
+     :alt: The Main Menu screen of the Collect app. The three-dot 'kebab' menu in the upper-right corner is circled in red.
+
+   .. image:: /img/collect-connect/kebab-menu-general-settings.*
+     :alt: The Main Menu screen of the Collect App. A modal menu has unrolled in the top-right corner, with the option *About*, *General Settings*, and *Admin Settings*. *General Settings* is circled in red.
+
+2. Select :guilabel:`Server`
+
+   .. image:: /img/collect-connect/general-settings-server.*
+     :alt: The General Settings menu in the Collect app. The options are *Server*, *User Interface*, *Form management*, and *User and device identity*. *Server* is circled in red.
+
+3. Make sure :guilabel:`ODK` is selected under :guilabel:`Type` and then fill in the :guilabel:`URL`, :guilabel:`Username` and :guilabel:`Password` for your server:
+
+   .. image:: /img/collect-connect/server-settings-odk.*
+     :alt: The Server Settings screen in the Collect app. Below the *Type* option there are three items labeled *URL*, *Username*, and *Password*.
+
 Other options
 ~~~~~~~~~~~~~~~
 
 .. toctree::
   :maxdepth: 1
 
   collect-connect-google
-  collect-connect-aggregate
```

Comment:
In `s3_website.yml`, add `collect-connect-aggregate/index.html: collect-connect`